README.md

```diff
@@ -0,0 +1,21 @@
+# whitespace-maven-plugin
+Maven plugin for removing trailing whitespace errors
+
+# Example Usage
```

Comment:
why are the two titles both at the same level? Compare to https://raw.githubusercontent.com/spring-projects/spring-boot/master/README.adoc where only the first heading is .h1